settings.py

```diff
@@ -437,7 +437,7 @@
 
 # after login, django redirects to this URL
 # rather than the default 'accounts/profile'
-LOGIN_REDIRECT_URL = '/'
+LOGIN_REDIRECT_URL = 'homepage'
```

Comment:
@nickpell I'm wondering if this has something to do with recent url changes. I always got an error locally on redirecting to `/`, but I think `homepage` is the same thing. Does that seem like a recent django change that could have happened?